RiotSharp/Misc/Converters/RegionConverter.cs

```diff
@@ -52,6 +52,24 @@ public override bool CanConvert(Type objectType)
                 case "JP":
                 case "jp":
                     return Region.Jp;
+                case "AMERICAS":
```

Comment:
not related to this PR, but can we do `ToLower()` and have only one case per region?